client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -67,36 +75,56 @@ class SelectBusinessType extends Component {
 					/>
 				</Card>
 
-				<CTACard
-					headerText={ translate( 'Physical Location or Service Area', {
-						comment: 'In the context of a business activity, brick and mortar or online service',
-					} ) }
-					mainText={ translate(
-						'My business has a physical location customers can visit, ' +
-							'or provides goods and services to local customers, or both.'
-					) }
-					buttonText={ translate( 'Create My Listing', {
-						comment: 'Call to Action to add a business listing to Google My Business',
-					} ) }
-					buttonIcon="external"
-					buttonPrimary={ true }
-					buttonHref="https://www.google.com/business/"
-					buttonTarget="_blank"
-					buttonOnClick={ this.trackCreateMyListingClick }
-				/>
+				<Card className="select-business-type__cta-card">
+					<div className="select-business-type__cta-card-main">
+						<h2>
+							{ translate( 'Physical Location or Service Area', {
+								comment:
+									'In the context of a business activity, brick and mortar or online service',
+							} ) }
+						</h2>
+						<p>
+							{ translate(
+								'My business has a physical location customers can visit, ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Your business has a physical location customers can visit, or provides goods and services to local customers, or both.' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).